app/controllers/webhooks_controller.rb

```diff
@@ -139,8 +140,9 @@ def handle_payment_intent_succeeded(
             seller_id: seller.seller_id,
             purchaser: purchaser,
             payment_intent: payment_intent,
-            amount: amount_per
+            amount: amount_per + (remainder > 0 ? 1 : 0)
```

Comment:
Do you know if @donation_sellers.each is deterministically ordered? If so, won't the remainder always go to the same businesses? I don't think that's a huge problem, but getting the cents to go to random sellers was the bit of complexity I was avoiding implementing in the first place.

Reply:
it's not a huge problem, and i don't think matters too much. i could shuffle the sellers, but that would be a nightmare to test, and i don't think they'd fret over the extra cent. If we get 300 pooled donations, it would be a difference of only three dollars

we can merge as is, and maybe we can follow up shuffling the merchants and writing a test for that shuffle

Reply:
sgtm